src/utils/json.ts

```diff
@@ -1,4 +1,37 @@
+// -------------------------------------------------------------
+// WARNING: this file is used by both the client and the server.
+// Do not use any browser or node-specific API!
+// -------------------------------------------------------------
+
 // NOTE: We should store the methods of the `JSON` object
 // since they can be overriden by the client code.
-export const parseJSON     = JSON.parse;
-export const stringifyJSON = JSON.stringify;
+export const parse     = JSON.parse;
+export const stringify = JSON.stringify;
+
+function isDOMNode (obj) {
+    if (typeof Node === 'object')
+        return obj instanceof Node;
+
+    return typeof obj.nodeType === 'number' && typeof obj.nodeName === 'string';
+}
+
+function isJQueryObj (obj) {
+    return !!(obj && obj.jquery);
+}
+
+export function isSerializable (value) {
```

Comment:
Because we decided to get rid of the `json-hammerhead` completely, I had to move these functions to our codebase. It's used in the `testcafe-legacy-api` package: https://github.com/DevExpress/testcafe-legacy-api/blob/60cd17a636359d14e4ffbec4316421e9ca1ab3b5/src/client/step-iterator.js#L69